From b0acad742eee36ed97767f43a17b549097ab0c9d Mon Sep 17 00:00:00 2001
From: Alex Deucher <alexdeucher@gmail.com>
Date: Wed, 24 Mar 2010 18:45:32 -0400
Patch-mainline: 2.6.34
References: fate#310916
Git-commit: d8fe2fa90ea9df694dcc022271391e7547772e1b
Subject: [PATCH 0535/2588] drm/radeon/r600: remove some regs are not safe
 regs for command buffers

Only the drm should be touching them.

Signed-off-by: Alex Deucher <alexdeucher@gmail.com>
Signed-off-by: Dave Airlie <airlied@redhat.com>
(cherry picked from commit d8fe2fa90ea9df694dcc022271391e7547772e1b)

Signed-off-by: Takashi Iwai <tiwai@suse.de>
---
 drivers/gpu/drm/radeon/reg_srcs/r600 |    3 ---
 1 files changed, 0 insertions(+), 3 deletions(-)

diff --git a/drivers/gpu/drm/radeon/reg_srcs/r600 b/drivers/gpu/drm/radeon/reg_srcs/r600
index 8f414a5..1d33400 100644
--- a/drivers/gpu/drm/radeon/reg_srcs/r600
+++ b/drivers/gpu/drm/radeon/reg_srcs/r600
@@ -824,8 +824,6 @@ r600 0x9400
 0x00028428 CB_FOG_GREEN
 0x00028424 CB_FOG_RED
 0x00008040 WAIT_UNTIL
-0x00008950 CC_GC_SHADER_PIPE_CONFIG
-0x00008954 GC_USER_SHADER_PIPE_CONFIG
 0x00009714 VC_ENHANCE
 0x00009830 DB_DEBUG
 0x00009838 DB_WATERMARKS
@@ -834,4 +832,3 @@ r600 0x9400
 0x00009504 TA_CNTL
 0x00009700 VC_CNTL
 0x00009718 VC_CONFIG
-0x0000A02C SMX_DC_MC_INTF_CTL
-- 
1.7.6

